### ISSUE TITLE
weird fonts/sizing in menus

### ISSUE THREAD

**Issue by shriram** [brownplt/code.pyret.org#342]
Hovering over the Pyret menu gives a tooltip in a font that is totally different from that used elsewhere on the page:
![image](https://user-images.githubusercontent.com/75030/92342918-56c61380-f090-11ea-8c46-a1654a4c78a6.png)

Also, the top entry has a line break for some reason instead of just being wide enough. It looks okay on Brave:
![image](https://user-images.githubusercontent.com/75030/92342934-66ddf300-f090-11ea-8c5b-adc62fa7bf8f.png)
but really not so much on Safari. I can't get the popup to stay up when I use the screenshot feature, but it looks like
```
Keyboard Shortcuts, Ctrl-
?
```

**Comment by blerner** [comment]
This is a papercut annoyance that's bothered me for a while: in general, we ought to be able to make every menu item be styled as `white-space: nowrap;`, which would solve this problem for all the menu items, but break the "Contribute detailed usage information" one.  I had "fixed" it partially for the theme menu item by stipulating that _that_ menu item not wrap, and on my machines, that was the widest menu item, so it "fixed" things in general.  I didn't know Safari was 

So maybe we do
```
#bonniemenu > li { white-space: nowrap; }
li#the-one-with-the-logging { white-space: unset; }
```
?

The font comes from https://github.com/brownplt/code.pyret.org/blob/horizon/src/web/css/shared.css#L551, or more accurately the _lack_ of font I think comes from there.  But I don't know if those tooltips are used elsewhere in CPO, so I don't know if it's a good idea to change it right there.

**Comment by asolove** [comment]
I would really love to understand the tooltip and either get rid of it (in favor of a better way to make the icon accessible) or at least fix it up so it doesn't cover up the menu after the icon is clicked to show the menu. Would folks be interested in a PR for that?

**Comment by jpolitz** [comment]
That would be great! I think this is some weirdness that was accidentally introduced when we added some accessibility features, so if you can identify a cleanup that would be terrific.

**Comment by asolove** [comment]
- The tooltip is removed in #374 
- The random pieces of text rendering in an unwanted font was fixed in #373

For the menu item line-breaks, I have a suggestion on updating the menu design and might fold changing text and preventing line breaks as part of that work.

**Comment by asolove** [comment]
I believe this can now be closed as fixes are in Horizon.
